src/components/TermsDialog.tsx

```diff
@@ -0,0 +1,113 @@
+import { Dialog, Transition } from '@headlessui/react';
+import clsx from 'clsx';
+import { Fragment, useState } from 'react';
+import { termsIndexAgreedUponAtom } from 'store/app';
+import { useSetAtom } from 'jotai';
+
+// Increment every time the current terms change.
```

Comment:
how will we ensure this happens? (in all affected repos)

not a blocker but I think it deserves a ticket for follow-up